src/lib/chimp-spec.js

```diff
@@ -0,0 +1,75 @@
+const Chimp = require('./chimp');
```

Comment:
why is this test in `src/lib/` while the other tests are in `src/__tests_`? should I be adding new tests to `src/lib` instead?